packages/wdio-mocha-framework/src/index.js

```diff
@@ -97,14 +95,24 @@ class MochaAdapter {
         const match = MOCHA_UI_TYPE_EXTRACTOR.exec(options.ui)
         const type = (match && INTERFACES[match[1]] && match[1]) || DEFAULT_INTERFACE_TYPE
 
+        const hookArgsFn = (context) => {
+            const test = context.test || {}
+            const parentTitle = test.parent ? test.parent.title : ''
+            return [{ ...test, parent: parentTitle || '' }, context]
```

Comment:
I think parentTitle will always exists, you initialized it line before. Or you could think that test.parent.title can not exist even if test.parent will exist?

Reply:
I think test.parent.title might not exist in some cases, but I'm not sure.
Have to check if test.parent.title value exists for root suite for both it and before

Reply:
@CrispusDH updated.